channels/tr.m3u

```diff
@@ -649,3 +649,5 @@ https://broadcasttr.com:446/oncurtv/bant1/playlist.m3u8
 https://5be5d840359c6.streamlock.net/unitv/unitv/chunklist.m3u8
 #EXTINF:-1 tvg-id="UUTVUskudarUniversitesiTV.tr" tvg-name="ÜÜ TV Üsküdar Üniversitesi TV" tvg-country="TR" tvg-language="" tvg-logo="" group-title="",ÜÜ TV Üsküdar Üniversitesi TV
 http://uskudarunv.mediatriple.net/uskudarunv/uskudar2/iptvdelisi.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Xəzər
+http://mfe.cliptv.az/dash/Xezer_SD.ism/playlist.mpd
```

Comment:
This is an Azerbaijani channel: https://az.wikipedia.org/wiki/X%C9%99z%C9%99r_TV